build/docker/debian/Dockerfile

```diff
@@ -113,8 +113,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
       neko-dev \
       libneko0
 
-# Node.js dependencies - THRIFT-4064 says it must be >= 4.x
-RUN curl -sL https://deb.nodesource.com/setup_4.x | bash -
+# Node.js dependencies - THRIFT-4064 says it must be >= 0.12.0
```

Comment:
Wasn't the whole pont of this exercise to move up to node 4.x as a minimum supported version?